app/scripts/controllers/permissions/enums.ts

```diff
@@ -2,4 +2,5 @@ export enum NOTIFICATION_NAMES {
   accountsChanged = 'metamask_accountsChanged',
   chainChanged = 'metamask_chainChanged',
   sessionChanged = 'wallet_sessionChanged',
+  walletNotify = 'wallet_notify',
```

Comment:
Will add `wallet_sessionChanged` and `wallet_notify` to `@metamask/multichain-api-middleware` package

Reply:
Will do here: https://github.com/MetaMask/core/pull/5491/files